### ISSUE TITLE
Twitter Social Monitoring not making counts

### ISSUE THREAD

**Issue by khaoua** [mautic/mautic#4772]
What type of report is this:

| Q  | A
| ---| ---
| Bug report? | yes
| Feature request? | 
| Enhancement? | 

## Description:
social monitoring for tweets created (both hashtags and mentions). Tested several tweets, with caps and no caps to see differences in case cap sensitive. Tested with several twitter accounts. Mautic acting weird, sometime contacts are added to twitter contact segment that was initially created, sometimes not. Sometimes it add contact tab in monitoring sometime not. Sometimes count goes in, but most of the time, no counts. 

All this happens although Twitter app is created, app is authorised and plugin active

any input on this? any missing check I might forgot or else

basically, I have created several twitter monitoring, but all of them are acting randomly without any logic, could be good to investigate, I'm open and available to test any proposed approach

ps: crons are set up with cpanel, run successfully but act as no tweets were added as value of cron result is always 0 for each monitoring item
## If a bug:

| Q   | A
| --- | ---
| Mautic version | 2.9.2
| PHP version | 

### Steps to reproduce:
1. 
2. 
 
### Log errors: 

_Please check for related errors in the latest log file in [mautic root]/app/log/ and/or the web server's logs and post them here. Be sure to remove sensitive information if applicable._

**Comment by khaoua** [comment]
some pictures to illustrate the issue

this monitoring below got several tweets, contact is known, but not shown in monitoring contact tab

![image](https://user-images.githubusercontent.com/25910708/30063134-de2d08b4-924d-11e7-86b3-b1736ff760b4.png)

this one below has several tweets, no count shows up, nor contact in contact tab

![image](https://user-images.githubusercontent.com/25910708/30063310-871ff904-924e-11e7-87c5-9685ed7ae508.png)

this one below is for a tweet mentions, same for the hashtags above, no count and no contact

![image](https://user-images.githubusercontent.com/25910708/30063374-d1124aa8-924e-11e7-8325-6c78af0c4b77.png)
